src/app/health-page/health-data.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { DspaceRestService } from '../core/dspace-rest/dspace-rest.service';
+import { RawRestResponse } from '../core/dspace-rest/raw-rest-response.model';
+import { HALEndpointService } from '../core/shared/hal-endpoint.service';
+
+@Injectable({
+    providedIn: 'root'
+})
+export class HealthDataService {
```

Comment:
I wouldn't call it HealthDataService if it doesn't have the `@dataService` decorator. To prevent confusion. 

Or even better, actually implement it as a dataservice. I can see these calls are very straightforward, so it would be a little overhead for this particular case, but the advantage is consistency.

Every other request the UI makes to our backend can be found in the request section of the ngrx store, except for this service. So at some point in the future someone will be debugging some problem and waste a bunch of time wondering why they see requests in the network tab that don't show up in the store.

Reply:
I agree we should avoid to not extending data service, but the main reason why we can't implement the data service is that the rest response come from a spring boot implementation and they don't have the `type` attribute. So we are not able to use the dataservice

Reply:
I see. In that case I'd just rename it to prevent confusion.